demo/src/vendor.ts

```diff
@@ -0,0 +1,6 @@
+import 'angular2/bundles/angular2-polyfills';
```

Comment:
I thought that we had a discussion about it :-) I need to fix it in all the other seed repos.